src/main/java/sirius/pasta/tagliatelle/tags/ArgTag.java

```diff
@@ -157,4 +159,9 @@ public Class<?> getExpectedAttributeType(String name) {
 
         return super.getExpectedAttributeType(name);
     }
+
+    @Override
+    public Set<String> getRequiredAttributeNames() {
+        return new HashSet<>(List.of(PARAM_NAME, PARAM_TYPE));
```

Comment:
**Question:** Why is `Set.of` not used here instead of creating a list and then turning that list into a set?

Reply:
The modifiable list is needed because the calling method is removing the recognized attributes. I redo the the methods in the tags with Set.of(...) and make the new HashSet(...) in the check method.